.github/workflows/ci.yml

```diff
@@ -301,13 +302,12 @@ jobs:
       contents: write
     steps:
       - name: "Release to the public PyPI repository"
-        uses: ansys/actions/release-pypi-public@v8
+        uses: ansys/actions/release-pypi-public@v9

```

Comment:
You need to reimplement this action following the migration guide @germa89 -- otherwise it is going to fail the next time you do a release

Reply:
I did use the migration guide (referenced in the issue I did for this #3831 

But maybe I missunderstood, I thought that if I do not use `trusted_publisher` I could use the standard composite action...  

Anyways #3837 should fix it.

Reply:
You were using the trusted-publisher approach -- you no longer had a dedicated token to upload to PyPI. Meaning, that you needed to migrate to the new method for publishing wheels using the trusted publisher. Thanks for opening #3837!